src/anshuKJun20/DuplicateNumber.java

```diff
@@ -0,0 +1,21 @@
+package anshuKJun20;
+
+public class DuplicateNumber {
+	void findDuplicateNumber(int[] arr) {
+		int count = 0;
+		for (int i = 0; i < arr.length; i++) {
+			for (int j = 0; j < arr.length; j++) {
+				if (arr[i] == arr[j]) {
+					count = arr[i];
```

Comment:
Here at line number 5 no need to initialize variable count.You can directly print duplicate number in print statement .
Ex. System.out.println("\nDuplicate Number is: " + input[i]);

Reply:
Actually what I was expecting is don't initialize any variable.As we are not returning or reusing duplicate number try to optimize it .
for (int i= 0; i< input.length; i++) {
			for (int j= i+1; j< input.length; j++) {
				if (input[i] == input[j]) {
					System.out.println("\nDuplicate Number is: " + input[i]);
					break;

* If you want to initialize variable then needs to be local variable.no need of instance variable.